keras_retinanet/preprocessing/pascal_voc.py

```diff
@@ -81,7 +81,7 @@ def __init__(
         self.data_dir             = data_dir
         self.set_name             = set_name
         self.classes              = classes
-        self.image_names          = [l.strip() for l in open(os.path.join(data_dir, 'ImageSets', 'Main', set_name + '.txt')).readlines()]
+        self.image_names          = [l.strip().split(None, 1)[0] for l in open(os.path.join(data_dir, 'ImageSets', 'Main', set_name + '.txt')).readlines()]
```

Comment:
I would prefer to write this as `str.split(maxsplit=1)` instead of `str.split(None, 1)`. I think the first version is much easier to understand without referring to the `str` documentation.

Also, what does the -1 even mean here? Why is it there?